practices/code-review.md

```diff
@@ -60,11 +60,30 @@ Here's the general process that we use:
 
 ## What to look out for
 
-* Are there syntactic inconsistencies the linter did not catch?
-* Overly complex code.
-* PRs which are non-atomic and should be split into separate PRs, e.g. a new UI feature combined with an unrelated configuration change.
-* Code which could be split up into reusable modules.
-* Non-performant code. Suggest testing with [WebPageTest](https://www.webpagetest.org/) and/or other tools.
-* Inaccessible code. Suggest testing with [pa11y](https://github.com/pa11y) and/or other tools.
+* **Accessibility**
+
+  * Accessibility problems. Suggest testing with [pa11y](https://github.com/pa11y) and/or other tools.
```

Comment:
You could link to the accessibility checklist here (or the house style)

Reply:
I forgot the links! 🤦